tests/fixtures/rust_project/function-item.scm

```diff
@@ -0,0 +1 @@
+(function_item) @function_item
```

Comment:
I meant to ask in some other PR, what's up with the "scm" extension on these files?

Reply:
[This](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#queries) explains it, I guess "Scheme" or (perhaps after-the-fact) "source code matching"